activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,46 @@
+require 'active_support/time'
+
+module ActiveSupport  
+  class Claims
+    class InvalidClaims < StandardError; end
+    class ExpiredClaims < StandardError; end
+    
+    attr_reader :payload, :purpose, :expires
+
+    def initialize(options)
```

Comment:
Are we still going with the keyword arguments? Do we want different arguments for payload and other options purpose and expiry like you wrote in the code above ? And if so, why is that ?